pom.xml

```diff
@@ -15,6 +15,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
Why is this needed?

Reply:
Actually it is not, at least not yet, i just deleted it and updated the POM https://github.com/algutierrez-griddynamics/e-commerce/pull/11/commits/3639dd11913cad56656ac7f5bc4b8d524ad40841, i also added some testing dependencies as well as some plugins https://github.com/algutierrez-griddynamics/e-commerce/pull/11/commits/8d58e474e91d7c02ae0296a744c2733d03d37ee8 and lombok https://github.com/algutierrez-griddynamics/e-commerce/pull/11/commits/1ff7a447bda373a2ad6a8786e839c97763e2b17c